Examples/MultiModal/Mixed_Image__And_Tabular_Reading/.ipynb_checkpoints/CompositeReader-checkpoint.ipynb

```diff
@@ -0,0 +1,429 @@
+    "\n",
+    "*You can think of a feature map as a set of features that you let the neural network figure out and tune on that are the best simplistic representation of the data which can be merged with other data sets.*\n",
+    "\n",
+    "We then take the feature map, which is essentially a numpy array and append the features from our ctf file, which is also essentially a numpy array.  These then form the input for a standard feed forward network to produce our 4 regressor outputs.\n",
+    "\n",
+    "*Notice that we use relu as the standard activation function, but on the final layer we specify an activation of None* a None activation essentially just computes a final linear regression on each output node with no extra function on top of it.  This allows you the ability to produce outputs from -infinity to +infinity.\n",
+    "\n",
+    "**Loss and Errors:**  Notice that we use squared_error as our loss.  This is because we are predicting regressors as our output and not a classifier.  errs = loss when you are predicting regressors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# function to build model\n",
+    "def create_model(x_i, x_t):\n",
+    "    with C.layers.default_options(init = C.layers.glorot_uniform(), activation = C.relu):\n",
+    "            h = x_i\n",
+    "            \n",
+    "            h = C.layers.Convolution2D(filter_shape=(5,5), num_filters=8, strides=(1,1), pad=True, name=\"conv_1\")(h)            \n",
+    "            h = C.layers.MaxPooling(filter_shape=(2,2), strides=(2,2), name=\"max_1\")(h)            \n",
+    "            h = C.layers.Convolution2D(filter_shape=(5,5), num_filters=16, strides=(1,1), pad=True, name=\"conv_2\")(h)            \n",
+    "            h = C.layers.MaxPooling(filter_shape=(3,3), strides=(3,3), name=\"max_2\")(h)\n",
+    "            \n",
+    "            # create a feature map\n",
+    "            h = C.layers.Dense(conv_feature_map_size, name=\"conv_feature_map\")(h)\n",
+    "            \n",
+    "            #merge the convolutional feature map with raw tabular data\n",
+    "            h = C.splice(h, x_t, axis=0)\n",
+    "            \n",
+    "            #mix up the data in a dense output sequence\n",
+    "            p = C.layers.Dense(num_regression_outputs, activation = None, name=\"prediction\")(h)\n",
+    "            \n",
+    "            return p\n",
+    "        \n",
+    "def create_errors(model, labels):\n",
+    "    loss = C.losses.squared_error(model, labels)\n",
+    "    errs = loss\n",
+    "    return loss, errs # (model, labels) -> (loss, error metric)        "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "## Training Progress Printer\n",
```

Comment:
Would prefer to use ProgressPrinter  framework. Refer to https://github.com/Microsoft/CNTK/blob/master/Manual/Manual_How_to_feed_data.ipynb for example